src/managing-workflow/tuning-component-settings.md

```diff
@@ -19,6 +19,23 @@ env:
     value: "admin_only"
 ```
 
+## Setting Resource limits
+
+You can set resource limits to Workflow components by modifying the template file `tpl/generate_params.toml`.
+This file has a section for each Workflow component to set a limit to any Workflow component just add limits_cpu, limits_memory
```

Comment:
need to add a period:

for each Workflow component.* To set a limit ...

Also can you add backticks around `limits_cpu` and `limits_memory`?